core/models/broadcasts.go

```diff
@@ -483,7 +471,7 @@ func (b *BroadcastBatch) CreateMessages(ctx context.Context, rt *runtime.Runtime
 }
 
 func (b *BroadcastBatch) updateTicket(ctx context.Context, db Queryer, oa *OrgAssets) error {
-	firstReplySeconds, err := TicketRecordReplied(ctx, db, b.TicketID, dates.Now())
+	firstReplyTime, err := TicketRecordReplied(ctx, db, b.TicketID, dates.Now())
```

Comment:
unrelated but go linter is upset about the name of a `time.Duration` variable having the word "seconds" in it